rust/benches/utils.rs

```diff
@@ -14,6 +14,21 @@ use std::{
     time::{Duration, Instant},
 };
 
+/// Returns true with the given probability (in percentage)
+#[allow(dead_code)]
+pub fn with_prob(prob: u16) -> bool {
```

Comment:
```suggestion
pub fn with_prob(prob: u8) -> bool {
```

Why is this a u16 (here and everywhere else)?